src/DataMigration/DataMigration/Cmdlets/InvokeDataMigrationCommand.cs

```diff
@@ -30,9 +30,23 @@ public class InvokeDataMigrationCommand : DataMigrationCmdlet, IDynamicParameter
         [Parameter(
           Mandatory = true,
           HelpMessage = "Command Type.")]
-        [PSArgumentCompleter("Complete")]
```

Comment:
@sichoudh as discussed offline, it's advised to continue to use `PSArgumentCompleter` is this allows users to provide additional values that may be supported without having to wait for a new iteration of the module to introduce them into the enum. Since this is a preview module, you can choose to use enums instead, but this will resurface when the module is transitioned to stable in the future.

Reply:
Will keep as-is for now